src/master-playlist-controller.js

```diff
@@ -235,6 +237,8 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     this.syncController_ = new SyncController();
 
+    this.decrypter_ = work(Decrypter);
```

Comment:
This is to limit the number of workers we are creating. We currently already have one worker for transmuxing. Creating the decrypter in each loader brings that number up to 3, but there really isnt any reason we can use the same worker since decryption doesn't need state. Also, since we always create an audio segment loader, but the common use case is muxed video and audio source, the audio loader is often not used, but would still create a wasted worker